pkg/mysqlcluster/cluster.go

```diff
@@ -47,29 +47,29 @@ type cFactory struct {
 
 	namespace string
 
-	client    kubernetes.Interface
-	myClient  ticlientset.Interface
-	podLister corelisters.PodLister
-	rec       record.EventRecorder
+	client   kubernetes.Interface
+	myClient ticlientset.Interface
+	rec      record.EventRecorder
+
+	orcClient orc.Interface
 
 	configHash string
 	secretHash string
 }
 
 // New creates a new cluster factory
 func New(cluster *api.MysqlCluster, opt *options.Options, klient kubernetes.Interface,
-	myClient ticlientset.Interface, ns string, rec record.EventRecorder,
-	podLister corelisters.PodLister) Interface {
+	myClient ticlientset.Interface, ns string, rec record.EventRecorder) Interface {
```

Comment:
I think that `ticlientset` is a leftover from the time the project was called titanium. It should be mysqlclient.